src/ch/fhnw/cpib/parser/AbsSyn.java

```diff
@@ -1402,6 +1406,8 @@ public int code(ICodeArray codeArray, int location, Environment env) throws Code
         public List<IDeclaration> globalDeclarations;
         public List<ICommand> commands;
 
+        private Map<String, VariableSignature> internalScope;
```

Comment:
das ha ich au scho in mim branch gmacht, wenns das do nid brucht wärs glaub eifacher es do wider usezneh

Reply:
Done